packages/frontend/src/components/UserSettings/AccessTokensPanel/index.tsx

```diff
@@ -145,12 +34,25 @@ const AccessTokensPanel: FC = () => {
                     title="No tokens"
                     description="You haven't generated any tokens yet!, generate your first token"
                 >
-                    <MantineButton onClick={() => setCreateInvitesPanel(true)}>
+                    <Button onClick={() => setCreateInvitesPanel(true)}>
                         Generate token
-                    </MantineButton>
+                    </Button>
                 </EmptyState>
-            )}
-        </Flex>
+            </Flex>
+        );
+    }
+
+    return (
+        <>
+            <Flex justify="space-between" align="center" mb="lg">
+                <Title order={5}>Personal access tokens</Title>
+                <Button onClick={() => setCreateInvitesPanel(true)}>
+                    Generate new token
+                </Button>
+            </Flex>
+
+            <TokensTable />
```

Comment:
I think modal should be rendered only once outside of the table (coming up in the next PR I guess)

Reply:
Great point, I put it inside `TokensTable` but on the same level as `Paper`. what do you think? Or should I move the modal to this view instead? I would prefer inside `TokensTable`

Reply:
sounds good